runtime/errors.go

```diff
@@ -62,9 +63,10 @@ var (
 	OtherErrorHandler = DefaultOtherErrorHandler
 )
 
-type errorBody struct {
-	Error string `json:"error"`
-	Code  int    `json:"code"`
+type ErrorBody struct {
```

Comment:
I don't think it is a good idea to expose this type just for testing.
This struct is just a convenient way for this `runtime` package to construct JSON object on error. But the struct itself is not a part of API exposed for users or generated codes by this package.